### ISSUE TITLE
Loop around

### ISSUE THREAD

**Issue by DadamaldaDad** [ZaksenCode/Oneblocked#2]
I can't find any code to loop around to the first stage after finishing the last stage. Is this implemented?

**Comment by ZaksenCode** [comment]
Nope. There is an item in the game to change the generator stage.

**Comment by DadamaldaDad** [comment]
Is there an automatic way to do this?

**Comment by ZaksenCode** [comment]
no
